.github/workflows/build.yml

```diff
@@ -6,6 +6,19 @@ on:
   pull_request:
 env:
   GO_VERSION: '1.17'
+permissions:
+  actions: none
+  checks: none
+  contents: none
+  deployments: none
+  id-token: none
+  issues: none
+  discussions: none
+  packages: none
+  pull-requests: none
+  repository-projects: none
+  security-events: none
+  statuses: none
```

Comment:
The documentation seems to imply that using `permissions:` without specifying each one of the options have the same effect as setting `none` to all of them.

I ended up choosing the verbose approach as it made explicit the permission model _intented_ without requiring awareness of this detail by whoever stumbles into this file.

https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#permissions